configs/rover_configs.yaml

```diff
@@ -37,30 +37,33 @@ core:
 process_discovery:
   # The period of report or keep alive process(second)
   heartbeat_period: ${ROVER_PROCESS_DISCOVERY_HEARTBEAT_PERIOD:20s}
-  # Detect processes in VM mode
-  vm:
-    # Is active the VM mode to detect processes
-    active: ${ROVER_PROCESS_DISCOVERY_VM_ACTIVE:false}
+  # Scan process from linux
+  scanner:
     # The period to detect the process
-    period: ${ROVER_PROCESS_DISCOVERY_VM_PERIOD:3s}
-    finders:
+    period: ${ROVER_PROCESS_DISCOVERY_SCAN_PERIOD:3s}
+    # The process detect mode of scanner, support "REGEX", "AGENT_SENSOR"
+    mode: ${ROVER_PROCESS_DISCOVERY_SCAN_MODE:AGENT_SENSOR}
+    agent:
+      # Supports scanning for how long ago of the active agent
+      recent: ${ROVER_PROCESS_DISCOVERY_AGENT_RECENT:10m}
+    regex:
       # Use regex string to locate the processes
       # Duplicate entities cannot be reported. If multiple entity are generated, only one process will be report
       # If the multiple finders could match the same one process, only the first finder could be selected and report
-      - match_cmd_regex: ${ROVER_PROCESS_DISCOVERY_VM_FINDER_MATCH_CMD_REGEX:}
+      - match_cmd: ${ROVER_PROCESS_DISCOVERY_REGEX_FINDER_MATCH_CMD_REGEX:}
```

Comment:
`FINDER` or `SCANNER`, could you make them consistent in the doc and env variable?

Reply:
Updated.